src/main/resources/META-INF/rewrite/mockito.yml

```
@@ -121,6 +122,30 @@ recipeList:
       newVersion: 3.x
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.testing.mockito.AnyToNullable
+displayName: Replace Mockito 1.x `any(Class)` and `anyString()` with `nullable(Class)`
+description: Since Mockito 2.10 `any(Class)` and `anyString()` no longer match null values. Use `nullable(Class)` instead.
+tags:
+  - testing
+  - mockito
+applicability:
+  anySource:
+    - org.openrewrite.maven.search.FindDependency:
+        groupId: org.mockito
+        artifactId: mockito-all
+#    - org.openrewrite.gradle.search.FindDependency:
+#        groupId: org.mockito
+#        artifactId: mockito-all
```

Curiously, when I enable this, AnyToNullableTest starts to fail, even though it's [within an anySource block](https://docs.openrewrite.org/reference/yaml-format-reference) 🤔 
> If the anySource list is specified, OpenRewrite will run every recipe in the anySource list on every file in the repository the recipe is being run on. If any of those recipes result in a change to at least one file, then the declarative recipe is considered "applicable" for that repository and the recipes specified in the recipeList will be run on every file in the repository.

```suggestion
    - org.openrewrite.gradle.search.FindDependency:
        groupId: org.mockito
        artifactId: mockito-all
```
Any idea why @kunli2 ?

---

We can of course limit it to what it is now, and only perform these recipes for Maven.
What I hope to prevent with these applicability tests is that we do not migrate calls from 2.x or 3.x to `nullable`, as we should not; only when we migrate from 1.x to 2.10+ should we migrate these calls, as per #319